hazelcast/src/test/java/com/hazelcast/query/impl/extraction/MultipleCollectionsReflectionExtractionTest.java

```diff
@@ -0,0 +1,53 @@
+package com.hazelcast.query.impl.extraction;
+
+import com.hazelcast.config.InMemoryFormat;
+import com.hazelcast.query.Predicates;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static com.hazelcast.query.impl.extraction.MultipleCollectionsDataStructure.Person;
+import static com.hazelcast.query.impl.extraction.MultipleCollectionsDataStructure.limb;
+import static com.hazelcast.query.impl.extraction.MultipleCollectionsDataStructure.person;
+
+
+@RunWith(Parameterized.class)
+public class MultipleCollectionsReflectionExtractionTest extends AbstractExtractionTest {
```

Comment:
You can run your parameterized tests with `@RunWith(HazelcastTestRunner.class)` and you should add category to your tests, otherwise they won't run in the PR builder and any other test profile.